src/app/api/peanut/get-link-info/route.ts

```diff
@@ -0,0 +1,28 @@
+import type { NextRequest } from 'next/server'
+import { NextResponse } from 'next/server'
+import * as consts from '@/constants'
+
+export async function POST(request: NextRequest) {
+    try {
+        const { link } = await request.json()
+
+        const response = await fetch(`${consts.PEANUT_API_URL}/get-link-info`, {
```

Comment:
comment(blocking): the existing endpoint is /get-link-details

Also where is used this new route? I don't see it used in this PR